app/javascript/react/components/PizzaForm.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, } from "react";
+
+const PizzaForm = (props) => {
+  const [newPizza, setNewPizza] = useState({
+    product_name: "",
+    cost: "",
+    brand_id: null
+  })
+
+  const handleChange = (event) => {
+    event.preventDefault()
+    setNewPizza({
+      ... newPizza,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  } 
+
+  const formSubmitCallback = (event) => {
+    event.preventDefault()
+
+    const tempPizza = newPizza
+    tempPizza.brand_id = props.brand_id
+    setNewPizza(tempPizza)
+
+    props.addPizza(newPizza)
+    handleClearForm()
+  }
+
+  const handleClearForm = () => {
+    setNewPizza({
+      product_name: "",
+      cost: "",
+      brand_id: null
+    })
+  }
+
+  const errorList = props.errors.map( error => {
+      return(<li>{error}</li>)
```

Comment:
Take a look at this article for a reusable ErrorList component
https://learn.launchacademy.com/lessons/react-form-validations-with-hooks